xml/items/centipedes_gland.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <Item xmlns="http://www.hybrasyl.com/XML/Hybrasyl/2020-02">
-  <Name>Centipede Gland</Name>
```

Comment:
This was changed to match staging, change the references instead of changing the name